app/api/datastore/maat/applications.rb

```diff
@@ -0,0 +1,23 @@
+module Datastore
+  module Maat
+    class Applications < Base
+      version 'maat', using: :path
```

Comment:
I'm not sure about using 'version' to namespace this endpoint. I think there might be a namespace method or consider moving this code to sit alongside the Datastore Module rather than within in. I think we need to be able to independently version this endpoint from the rest of the datastore.